actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -88,13 +92,15 @@
 
     private CompletableFuture<Void> hostingActive = new Task<>();
 
-    private int maxLocalAddressCacheCount = 10_000;
-
     private NodeSelectorExtension nodeSelector;
 
-    public Hosting()
+    public Hosting(final int maxLocalAddressCacheCount, final long defaultActorTTL)
```

Comment:
In our use case, we would end up setting the max size some some absurdly large number anyway, which would make the cache evictions primarily based on expireAfterAccess.  Having the max size, to me is an insurance policy that you have some way of limiting the memory it can possibly use.  I think having both is still valid, but the tuning of the params is important to understand